pages.tr/linux/pacman-mirrors.md

```diff
@@ -0,0 +1,25 @@
+# pacman-mirrors
+
+> Manjaro Linux için pacman aynalistesi oluşturucu.
+> pacman-mirrors'ın çalıştırıldığı her vakit, E`sudo pacman -Syyu` komutu ile veritabanının senkronize edilmesi ve sistemin güncellenmesi gerekir.
+> Daha fazla bilgi için: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
```

Comment:
is `Daha fazla bilgi` a more exact translation from `More information`?
```suggestion
> Daha fazla bilgi: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
```

Reply:
`Daha fazla bilgi için` makes more sense in Turkish language.